CHANGES.rst

```diff
@@ -1,6 +1,14 @@
 Changes
 =======
 
+TBR
+------------------
+
+* The AutoExtract provider now supports the new providers API for caching
```

Comment:
`scrapy_autoextract.cache.AutoExtractCache` and `scrapy_poet.cache.SqlitedictCache` are quite the same.

Should we update `scrapy_autoextract.cache.AutoExtractCache` to effectively utilize the existing `scrapy_poet.cache.SqlitedictCache` code? or perhaps even deprecate it completely?

Reply:
After some discussions, it's been agreed that we can refactor `AutoExtractCache`'s implementation to use the new one provided by `scrapy-poet`.

This can be tackled further in https://github.com/scrapinghub/scrapy-autoextract/issues/32.